AK/SIMDExtras.h

```diff
@@ -49,6 +49,12 @@ ALWAYS_INLINE static u32x4 to_u32x4(TSrc v)
     return __builtin_convertvector(v, u32x4);
 }
 
+template<typename TSrc>
+ALWAYS_INLINE static i8x4 to_i8x4(TSrc v)
+{
+    return __builtin_convertvector(v, i8x4);
+}
+
```

Comment:
#24909 will replace that, with a nicer generic version

Reply:
Given time stamps on PRs, I think this one gets right of way :)